ipc/cli/src/commands/wallet/balances.rs

```diff
@@ -50,16 +49,19 @@ impl CommandLineHandler for WalletBalances {
 
                 let v: Vec<anyhow::Result<(TokenAmount, &EthKeyAddress)>> = join_all(r).await;
 
-                for r in v.into_iter().filter_map(|r| r.ok()) {
-                    let (balance, addr) = r;
-                    if addr.to_string() != "default-key" {
-                        println!("{} - Balance: {}", addr.to_string(), balance);
-                        no_balance.retain(|a| a != addr);
+                for r in v.into_iter() {
+                    match r {
+                        Ok(i) => {
+                            let (balance, addr) = i;
+                            if addr.to_string() != "default-key" {
```

Comment:
What is this `"default-key"`?

Reply:
This is the internal key used by the wallet store to tag the address being used as the default one for the wallet.

Reply:
It's strange to see that returned in the API :thinking: 
And why do we skip printing the balance? Aren't you interested in seeing the balance of your default (maybe only) address? In JSON this could be `{addr: "f410...", balance: 1.23, default: true}`

Reply:
I believe the default still stands as its own entry, no? At least I still see my default address printed out.